sdk/core/azure-core/inc/azure/core/md5.hpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Utility functions to help compute the MD5 hash value for the input binary data.
+ */
+
+#pragma once
+
+#include <cstdint>
+#include <vector>
+
+namespace Azure { namespace Core {
+
+  /**
+   * @brief Represents the class for the MD5 hash function which maps binary data of an arbitrary
+   * length to small binary data of a fixed length.
+   */
+  class Md5 {
+  public:
+    /**
+     * @brief Construct a default instance of @Md5.
```

Comment:
Does `@Md5` works? Don't we have to use `#Azure::Core::Md5` ?

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-cpp/pull/1647